### ISSUE TITLE
Using "skip to content" button on search results page clears result counts (staging only)

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse#3940]
## Description

During keyboard navigation, using the "skip to content" button on the search page, the page will lose all result counts and say "no results" for all media types.

## Reproduction

1.  Navigate to https://staging.openverse.org
2.  Tab once and select "skip to content"
3. Tab again to enter the search box
4. Type a search term ("galah" for example) and then enter to start the search
5. Allow the page to finish loading the results and confirm that there are results and that the result counts on the page reflect the number of results (for galah, that's >10k images and 8 audio). To do so using keyboard navigation and a screen reader, you can tab to the results list _without selecting skip to content_ and read the result counts.
6. If you tabbed to the results list, tab back to the skip to content button, and select it. If you are on the page without having interacted, tab once and select skip to content.
7. There will now be "no results" in all places that previously had result counts.

## Additional context

This is on staging only right now (I cannot reproduce on `openverse.org` only `staging.openverse.org`), but I am marking as critical because we cannot deploy the frontend with this bug.
